### ISSUE TITLE
final peer review

### ISSUE THREAD

**Issue by AriaYuqingXu** [PPPSDavid/ORIE-4741-Project#12]
Summary:
The project is about investigating factors behind flight delay problems. Their objective is to develop a model to predict the expected delay time for U.S. air traffic from 2016 to 2020. And they are combining two main datasets: Airline reporting carrier on-time performance dataset that contains flight information and US Weather Events dataset which contains weather warnings issued by weather stations in different airports. And they choose to use random forests to be the final model to predict whether a flight will be delayed.

Things I liked:
I like the formatting and outline of the report because it is easy to read and follow. And I like how you divide each part into little sections with a lot of detail.
I also like the feature engineering part and how you analyze different features and provide some detailed explanation of what each of them represents and its values.
And I really like how you try different methods to improve the F1 scores and include a table to compare the performance of different algorithms you used.
Suggestions:
It would be better if you include both training and validation accuracy and how the accuracy changes with different sets of hyperparameters.
You can try to remove some of the less important features. It might help improve the model. 
You may also try Light GBM. It is a fast, high-performance gradient boosting framework based on a decision tree algorithm, used for classification and many other machine learning tasks, which is also worth trying. And since the dataset is really large, XGBOOST would take a long time to train. Light GBM has Faster training speed and higher efficiency.
